new.out

```diff
@@ -0,0 +1,16 @@
+Creating 500000 1-sample series with 1ms interval in /var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/BenchmarkQuerySelect1000000SeriesWith1Samples307014079/001/0
+Creating 500000 1-sample series with 1ms interval in /var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/BenchmarkQuerySelect1000000SeriesWith1Samples307014079/001/1
+goos: darwin
+goarch: arm64
+pkg: github.com/thanos-io/thanos/pkg/query
+BenchmarkQuerySelect/1000000SeriesWith1Samples/select-8  	       2	 781464270 ns/op
+Creating 50000 50-sample series with 1ms interval in /var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/BenchmarkQuerySelect100000SeriesWith100Samples3301505/001/0
+Creating 50000 50-sample series with 1ms interval in /var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/BenchmarkQuerySelect100000SeriesWith100Samples3301505/001/1
+BenchmarkQuerySelect/100000SeriesWith100Samples/select-8 	      15	  76778933 ns/op
+Creating 1 5000000-sample series with 1ms interval in /var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/BenchmarkQuerySelect1SeriesWith10000000Samples1509504691/001/0
+Creating 1 5000000-sample series with 1ms interval in /var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/BenchmarkQuerySelect1SeriesWith10000000Samples1509504691/001/1
+BenchmarkQuerySelect/1SeriesWith10000000Samples/select-8 	      85	  13045786 ns/op
+PASS
+ok  	github.com/thanos-io/thanos/pkg/query	12.864s
```

Comment:
Oops, this looks like it was committed accidentally (:

Reply:
Ouch, should be gone now :)